test/index.test.ts

```
@@ -197,4 +197,44 @@ describe("destr", () => {
+      { input: "-01", output: "-01" },
+      { input: "00", output: "00" },
+    ];
+
+    for (const testCase of testCases) {
+      expect(destr(testCase.input)).toStrictEqual(testCase.output);
+    }
+  });
+
+  it("still parses safe integer strings as numbers", () => {
+    const testCases: Array<{ input: string; output: unknown }> = [
+      { input: "9007199254740991", output: 9007199254740991 },
```

ESLint

[error] Invalid group length in numeric value.

(unicorn/numeric-separators-style)